<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>156</red>
             <green>27</green>
             <blue>145</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>156</red>
             <green>27</green>
             <blue>145</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕 Semilight</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>네이버 자동완성 키워드 추출기</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>검색어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyword"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="state">
       <property name="text">
        <string>상태메시지</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="start_btn">
           <property name="text">
            <string>추출시작</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_btn">
           <property name="text">
            <string>리셋</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_btn">
           <property name="text">
            <string>저장</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_btn">
           <property name="text">
            <string>종료</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
